content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -563,13 +563,13 @@ Validation Error Count: 1
 
 ## <inlinecode>db</inlinecode>
 
-> **Note**: `db push` and `db seed` are currently in [Preview](../../../about/releases#preview). Features in Preview may still have some issues, and are not recommended for use in production environments. To access commands in Preview, you must opt-in via the `--preview-feature` flag.
+> **Note**: `db seed` is currently in [Preview](../../../about/releases#preview). Features in Preview may still have some issues, and are not recommended for use in production environments. To access commands in Preview, you must opt-in via the `--preview-feature` flag.
```

Comment:
This note can probably be moved to `seed` then instead of being on this level?

Reply:
Good catch. Will move